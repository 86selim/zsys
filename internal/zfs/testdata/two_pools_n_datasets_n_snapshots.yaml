pools:
  - name: rpool
    datasets:
      - name: ROOT
      - name: ROOT/ubuntu
        zsys_bootfs: yes
        last_used: 2019-04-18T02:45:55+00:00
        mountpoint: /
        canmount: on
        snapshots:
          - name: snap_r1
            zsys_bootfs: yes:local
            mountpoint: /:local
            canmount: on:local
          - name: snap_r2
            zsys_bootfs: yes:local
            mountpoint: /:local
            canmount: on:local
  - name: bpool
    datasets:
      - name: BOOT
      - name: BOOT/boot
        zsys_bootfs: yes
        last_used: 2019-04-18T02:45:55+00:00
        mountpoint: /boot
        canmount: on
        snapshots:
          - name: snap_b1
            zsys_bootfs: yes:local
            mountpoint: /boot:local
            canmount: on:local
          - name: snap_b2
            zsys_bootfs: yes:local
            mountpoint: /boot:local
            canmount: on:local
