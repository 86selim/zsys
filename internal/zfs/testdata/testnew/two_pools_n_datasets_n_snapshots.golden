[
   {
      "Name": "bpool",
      "Mountpoint": "/",
      "CanMount": "off"
   },
   {
      "Name": "bpool/BOOT",
      "Mountpoint": "/BOOT",
      "CanMount": "off"
   },
   {
      "Name": "bpool/BOOT/boot",
      "Mountpoint": "/boot",
      "CanMount": "on",
      "BootFS": true,
      "LastUsed": 1555555555
   },
   {
      "Name": "bpool/BOOT/boot@snap_b1",
      "IsSnapshot": true,
      "Mountpoint": "/boot",
      "CanMount": "on",
      "BootFS": true,
      "LastUsed": 2000000000
   },
   {
      "Name": "bpool/BOOT/boot@snap_b2",
      "IsSnapshot": true,
      "Mountpoint": "/boot",
      "CanMount": "on",
      "BootFS": true,
      "LastUsed": 2000000000
   },
   {
      "Name": "rpool",
      "Mountpoint": "/",
      "CanMount": "off"
   },
   {
      "Name": "rpool/ROOT",
      "Mountpoint": "/ROOT",
      "CanMount": "off"
   },
   {
      "Name": "rpool/ROOT/ubuntu",
      "Mountpoint": "/",
      "CanMount": "on",
      "BootFS": true,
      "LastUsed": 1555555555
   },
   {
      "Name": "rpool/ROOT/ubuntu@snap_r1",
      "IsSnapshot": true,
      "Mountpoint": "/",
      "CanMount": "on",
      "BootFS": true,
      "LastUsed": 2000000000
   },
   {
      "Name": "rpool/ROOT/ubuntu@snap_r2",
      "IsSnapshot": true,
      "Mountpoint": "/",
      "CanMount": "on",
      "BootFS": true,
      "LastUsed": 2000000000
   }
]